utilities/boostHistHelpers.py

```diff
@@ -390,3 +393,36 @@ def set_flow(h, val="nearest"):
             nearest_vals = np.take(h.values(flow=True), -2, i) 
             np.take(h.values(flow=True), -1, i)[...] = nearest_vals if val == "nearest" else np.full_like(nearest_vals, val)
     return h
+
+def applyCorrection(h, scale=1.0, offsetCorr=0.0, corrFile=None, corrHist=None, createNew=False):
```

Comment:
I don't think this should be in boostHistHelpers, that was meant to be for utilities manipulating boost histograms. I don't know where it should go but somewhere in wremenants I think.